Intersect.Server/Entities/Entity.cs

```diff
@@ -1753,6 +1753,54 @@ public virtual void TryAttack(Entity target)
             );
             }
 
+            //Check on each attack if the enemy is a player AND if he is blocking.
```

Comment:
> //Check on each attack if the enemy is a player AND if he is blocking.

* use singular third-person pronoun:
_//Check on each attack if the enemy is a player AND if **they** are blocking._

source: https://apastyle.apa.org/blog/singular-they